library/src/test/java/com/chuckerteam/chucker/internal/support/JsonConverterTest.kt

```diff
@@ -32,6 +33,27 @@ class JsonConverterTest {
         assertThat(dateTestClass).isEqualTo(DateTestClass(Date(0)))
     }
 
+    @Test
+    fun testMoshiConfiguration_willSerialize_null_DateTime() {
+        val dateTestJson = JsonConverter.instance.adapter(DateTestClass::class.java)
+            .toJson(DateTestClass(null))
```

Comment:
Why you decided to create this test? Please, explain.

Reply:
Removed code related to `Date` parsing as it is not used anywhere in the codebase.